readium/lcp/src/main/java/org/readium/r2/lcp/LcpError.kt

```diff
@@ -21,6 +21,9 @@ public sealed class LcpError(
     override val cause: Error? = null
 ) : Error {
 
+    public object MissingCredentials :
```

Comment:
If it's in `LcpError` and not a shared `ContentProtection` error, I would keep with the LCP terminology and call it `MissingPassphrase`.